packages/data/src/api/metadataByMetadataId/metadataByMetadataId.test.ts

```diff
@@ -0,0 +1,35 @@
+import { GraphQLClient } from 'graphql-request';
+import { metadataByMetadataId } from './metadataByMetadataId';
+import { metadataByMetadataIdMock } from './metadataByMetadataId.mock';
+import { MetadataByMetadataIdQueryResult } from './metadataByMetadataId.types';
+
+jest.mock('graphql-request');
+
+describe('getMetadataByMetadataId', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'error').mockImplementation(() => {
+      // console.log('Suppressed console error.');
+    });
+  });
+
+  it('returns data', async () => {
+    (GraphQLClient as jest.Mock).mockImplementationOnce(() => ({
+      request: (): Promise<MetadataByMetadataIdQueryResult> =>
+        Promise.resolve(metadataByMetadataIdMock),
+    }));
+    const result = await metadataByMetadataId('test' );
```

Comment:
const result = await metadataByMetadataId('test' );
```suggestion
    const result = await metadataByMetadataId('test');
```